dpctl/tests/test_sycl_device.py

```diff
@@ -68,5 +68,21 @@ def test_get_max_num_sub_groups (self):
             self.fail("Encountered an exception inside get_max_num_sub_groups().")
         self.assertNotEqual(max_num_sub_groups, 0)
 
+    def test_get_aspects_base_atomics (self):
+        q = dpctl.get_current_queue()
+        try:
```

Comment:
Aha now I see. Basically we are testing that the function should never raise an exception. OK.